src/sentry/integrations/jira/integration.py

```diff
@@ -592,12 +586,19 @@ def get_create_issue_config(self, group, user, **kwargs):
         dynamic_fields = list(issue_type_meta["fields"].keys())
         dynamic_fields.sort(key=lambda f: anti_gravity.get(f) or 0)
 
+        # Only get the organization in the DB if there are dynamic_fields.
+        slug = None
         # build up some dynamic fields based on required shit.
         for field in dynamic_fields:
             if field in standard_fields or field in [x.strip() for x in ignored_fields]:
                 # don't overwrite the fixed fields for the form.
                 continue
-            mb_field = self.build_dynamic_field(group, issue_type_meta["fields"][field])
+
+            # TODO Organization can be null?
+            if not slug:
+                slug = Organization.objects.get(id=self.organization_id).name
```

Comment:
@mgaeta we should probably use the `get_from_cache` method here since slugs don't change very often.